core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -1020,6 +1020,12 @@ ExInstWithCause<CalciteException> failedToAccessField(
   @BaseMessage("No operator for ''{0}'' with kind: ''{1}'', syntax: ''{2}'' during JSON deserialization")
   ExInst<CalciteException> noOperator(String name, String kind, String syntax);
 
+  @BaseMessage("Invalid input for position function: from operand value must not be zero")
```

Comment:
BQ and Oracle both call this `position`, not `from`. Is there another dialect that calls it `from`?

Reply:
in the standard syntax in calcite it was Position(a IN b FROM c) so the c parameter was referred to as from in a couple spots. Decided to keep that especially in the function logic so there isn't a bunch of position variables in the position function.